java/src/main/java/ai/rapids/cudf/BinaryOperable.java

```diff
@@ -435,4 +435,12 @@ default ColumnVector shiftRightUnsigned(BinaryOperable shiftBy, DType outType) {
   default ColumnVector shiftRightUnsigned(BinaryOperable shiftBy) {
     return shiftRightUnsigned(shiftBy, implicitConversion(this, shiftBy));
   }
+
+  /**
+   * Calculate the log with the specified base, output is the same type as input.
+   */
+  default ColumnVector log(double base) {
```

Comment:
Please pass in the Scalar instead of creating it on the fly.  Also All scalars need to be closed.

Reply:
fixed